goldorak_simulation/config/control.yaml

```diff
@@ -0,0 +1,41 @@
+# This file contains configuration for the RazBot controllers. These should be
+# the same for simulated and live robots.
+
+# The joint state controller handles publishing transforms for any moving joints
+goldorak_joint_state_controller:
+  type: "joint_state_controller/JointStateController"
+  publish_rate: 50
+
+# The diff drive controller can control a Differential Drive or Skid Steer vehicle
+# with any arbitrary number of wheels.
+goldorak_diff_drive_controller:
+  type: "diff_drive_controller/DiffDriveController"
+  publish_rate: 50
+
+  left_wheel: 'left_wheel_joint'
+  right_wheel: 'right_wheel_joint'
+
+  wheel_separation: 0.194
+
+  # Odometry covariances for the encoder output of the robot. These values should
+  # be tuned to your robot's sample odometry data, but these values are a good place
+  # to start
+  pose_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
+  twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
+
+  # Top level frame (link) of the robot description
+  base_frame_id: base_link
+
+  # Velocity and acceleration limits for the robot
+  linear:
+    x:
```

Comment:
These are implemented as clamps on the simulated controller side.
It's not used by the "real controller" on the robot, although we could benefit from implementing clamps.

The others in `goldorak_navigation/param/base_local_planner_params.yaml` are limitations imposed on the planner that model the dynamics of the system more accurately: static friction is modeled with the minimum velocity applicable.